<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostEntry</class>
 <widget class="QDialog" name="PostEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>岗位录入</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color: rgb(255, 255, 255);
border-radius: 10px;
}
</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>481</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Microsoft JhengHei UI Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>岗位录入</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_1">
        <property name="font">
         <font>
          <family>Microsoft JhengHei UI Light</family>
         </font>
        </property>
        <property name="text">
         <string>支持文本、PDF、DOCX、图片（JPEG/PNG）录入</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="button_SelectFile">
        <property name="font">
         <font>
          <family>Microsoft JhengHei UI Light</family>
         </font>
        </property>
        <property name="text">
         <string>选择文件</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="placeholderText">
       <string>输入文本或拖拽文件到输入框内</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="button_input">
        <property name="font">
         <font>
          <family>Microsoft JhengHei UI Light</family>
         </font>
        </property>
        <property name="text">
         <string>录入</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="button_showMainWin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei UI Light</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">* {
  border-radius: 15px;
  border: none;
  background-color: rgb(255, 255, 255);
  text-align: left;
}</string>
   </property>
   <property name="text">
    <string>&lt; 返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
